contrib/hadolint.json

```diff
@@ -0,0 +1,598 @@
+            "const": "SC1098",
+            "description": "Quote/escape special characters when using `eval`, e.g. `eval \"a=(b)\"`.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC1098"
+          },
+          {
+            "const": "SC1099",
+            "description": "You need a space before the `#`.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC1099"
+          },
+          {
+            "const": "SC2002",
+            "description": "Useless cat. Consider `cmd < file | ..` or `cmd file | ..` instead.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2002"
+          },
+          {
+            "const": "SC2015",
+            "description": "Note that `A && B || C` is not if-then-else. C may run when A is true.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2015"
+          },
+          {
+            "const": "SC2026",
+            "description": "This word is outside of quotes. Did you intend to 'nest '\"'single quotes'\"' instead'?",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2026"
+          },
+          {
+            "const": "SC2028",
+            "description": "`echo` won't expand escape sequences. Consider `printf`.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2028"
+          },
+          {
+            "const": "SC2035",
+            "description": "Use `./*glob*` or `-- *glob*` so names with dashes won't become options.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2035"
+          },
+          {
+            "const": "SC2039",
+            "description": "In POSIX sh, something is undefined.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2039"
+          },
+          {
+            "const": "SC2046",
+            "description": "Quote this to prevent word splitting",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2046"
+          },
+          {
+            "const": "SC2086",
+            "description": "Double quote to prevent globbing and word splitting.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2086"
+          },
+          {
+            "const": "SC2140",
+            "description": "Word is in the form `\"A\"B\"C\"` (B indicated). Did you mean `\"ABC\"` or `\"A\\\"B\\\"C\"`?",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2140"
+          },
+          {
+            "const": "SC2154",
+            "description": "var is referenced but not assigned.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2154"
+          },
+          {
+            "const": "SC2155",
+            "description": "Declare and assign separately to avoid masking return values.",
+            "$comment": "https://github.com/koalaman/shellcheck/wiki/SC2155"
+          },
+          {
+            "const": "SC2164",
```

Comment:
Does this list contains all of the shellcheck possible errors?

Reply:
No. Just the ones from the readme. They are inherited from the previous iteration from schemastore.org.
I wonder if they should be in here at all.